internal/revproxy/main.go

```diff
@@ -67,7 +66,8 @@ func (r *Revproxy) RegisterHandlers(e *echo.Echo, commonMiddlewares ...echo.Midd
 	sk.GET("/", echo.NotFoundHandler)
 
 	// Routing for Renku services
-	e.Group("/api/notebooks", append(commonMiddlewares, notebooksRenkuAccessToken, notebooksRenkuRefreshToken, notebooksRenkuIDToken, notebooksGitlabAccessToken, notebooksAnonymousID(r.sessions), noCookies, stripPrefix("/api"), notebooksProxy)...)
+	// Notebooks is being routed to data service now
+	e.Group("/api/notebooks", append(commonMiddlewares, notebooksRenkuAccessToken, notebooksRenkuRefreshToken, notebooksRenkuIDToken, notebooksGitlabAccessToken, notebooksAnonymousID(r.sessions), noCookies, regexRewrite("^/api/notebooks(.*)", "/api/data/notebooks$1"), dataServiceProxy)...)
```

Comment:
Should we keep the same middleware stack as for `data-services`?


```suggestion
	e.Group("/api/notebooks", append(commonMiddlewares, renkuAccessToken, dataGitlabAccessToken, notebooksRenkuRefreshToken, notebooksAnonymousID(r.sessions), regexRewrite("^/api/notebooks(.*)", "/api/data/notebooks$1"), dataServiceProxy)...)
```

Reply:
I am not sure that the anonymous sessions in the old notebooks code for v1 sessions will work if we use the same middlewares as for v2.

Reply:
If it relies on `notebooksAnonymousID`, it should work.

Reply:
Yes now this has the same middleware as for data service (except for the path rewrite for notebooks which is necessary).